inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -82,11 +82,14 @@ bool getParameters(const pugi::xml_node& node, const std::string& name, std::vec
 }
 
 template <class T>
-bool stringToType(const std::string& valStr, T& value) {
+std::pair<T, bool> stringToType(const std::string& valStr) {
```

Comment:
Seems that this bool return value is not used at all, why not to remove it entirely?

Reply:
Boolean value removed.